### PR TITLE
Fix schema error

### LINKED ISSUE #57: Missing schema when changing theme
This has been a persistent issue when installing the themes in development mode and through `pip` in a clean conda environment.

As soon as the user changes the theme to `Pitaya Smoothie, ` the terminal throws the following missing schema error.

```bash
[W 2023-06-27 12:59:02.352 ServerApp] 404 GET /lab/api/settings/jupyterlab-accessible-themes:pitayasmoothie?1687867142302 (::1): Schema not found: /Users/tania/mambaforge/envs/jat-pip-installed/share/jupyter/lab/schemas/jupyterlab-accessible-themes/pitayasmoothie.json
[W 2023-06-27 12:59:02.352 LabApp] wrote error: 'Schema not found: /Users/tania/mambaforge/envs/jat-pip-installed/share/jupyter/lab/schemas/jupyterlab-accessible-themes/pitayasmoothie.json'
    Traceback (most recent call last):
      File "/Users/tania/mambaforge/envs/jat-pip-installed/lib/python3.11/site-packages/tornado/web.py", line 1784, in _execute
        result = method(*self.path_args, **self.path_kwargs)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/tania/mambaforge/envs/jat-pip-installed/lib/python3.11/site-packages/tornado/web.py", line 3290, in wrapper
        return method(self, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/tania/mambaforge/envs/jat-pip-installed/lib/python3.11/site-packages/jupyterlab_server/settings_handler.py", line 48, in get
        result, warnings = get_settings(
                           ^^^^^^^^^^^^^
      File "/Users/tania/mambaforge/envs/jat-pip-installed/lib/python3.11/site-packages/jupyterlab_server/settings_utils.py", line 370, in get_settings
        schema, version = _get_schema(schemas_dir, schema_name, overrides, labextensions_path)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/tania/mambaforge/envs/jat-pip-installed/lib/python3.11/site-packages/jupyterlab_server/settings_utils.py", line 47, in _get_schema
        raise web.HTTPError(404, notfound_error % path)
```

This indicates something wrong when registering the themes and generating the corresponding metadata.

## Steps to reproduce

1. Create a brand new conda environment
```bash
conda create -n jat-pip-installed -c conda-forge jupyterlab=3
conda activate jat-pip-installed
pip install jupyterlab-accessible-themes
jupyter lab
```
2. Head to JupyterLab > Settings > Pitaya Smoothie

### PR DESCRIPTION
**Pull request by steff456**
Fixes #57

This PR,

- [x] Removes unnecessary lerna option
- [x] Removes schema from main `package.json`
- [x] Removes schema folder from main package
- [x] Avoid setting registry inside the plugin of each theme

### CODE REVIEW

**Review by trallard — APPROVED** [review]
Tested locally and  it seems to have fixed the schema - will ask for a review from @gabalafou on the code side

File: packages/githublight/src/index.ts
Comment (gabalafou) [inline packages/githublight/src/index.ts:45]: I'm guessing this is the line that triggered the error in the server log. Is that correct?

Why was this code here to begin with? Was the idea that each theme could have configurable settings in the settings editor?

**Review by gabalafou — CHANGES_REQUESTED** [review]
I just have a few questions before I feel comfortable hitting the approve button

File: lerna.json
Comment (steff456) [reply lerna.json:9]: This ended up as a cleanup, from what I understand of [lerna's quickstart](https://lerna.js.org/docs/getting-started) we need to define the `workspaces` for it to work.

In multiple guides like this one, https://www.honeybadger.io/blog/monorepo-yarn-workspace-lerna/ and other projects I've seen the `workspaces` duplicate. I can try to remove it, but it will probably break the monorepo.

File: packages/githublight/src/index.ts
Comment (steff456) [reply packages/githublight/src/index.ts:45]: Yes, this was calling the endpoint for settings which as we don't handle raised the schema error we were seeing. This was part of the template from the cookiecutter and I also saw a couple of other extensions (that presumably) use settings so at the beginning I decided to leave it there. 

If in the future we want to have some configurations, we may need to add this code. For now, we can just remove it.

File: lerna.json
Comment (gabalafou) [reply lerna.json:9]: yeah it's just that the docs say that [Lerna relies on the workspaces setting defined for the package manager](https://lerna.js.org/docs/api-reference/configuration#packages) (i.e., npm, yarn)

Comment (gabalafou) [reply lerna.json:9]: Plus the Honeybadger blog, if I'm reading it correctly, instructs the reader to put workspaces in package.json not lerna.json

File: packages/githublight/src/index.ts
Comment (gabalafou) [reply packages/githublight/src/index.ts:45]: That makes sense. Thanks for the context!

**Review by gabalafou — APPROVED** [review]
I'm going to approve this so you're not blocked, but would you mind removing the workspaces key from lerna.json if you can? (or leave a comment if you can't)

**Comment by steff456** [comment]
I was able to remove those lines. I think this PR is ready for merge!

**Review by gabalafou — APPROVED** [review]

**Comment by trallard** [comment]
Linting, links, and formatting issues resolved, so ready to merge 🚀

**Pull request merged**
